ttnn/cpp/ttnn/tensor/layout/tensor_layout.hpp

```diff
@@ -31,6 +31,13 @@ class TensorLayout {
         const PageConfig& page_config,
         const MemoryConfig& memory_config,
         const ttnn::Shape& legacy_shape);
+    [[deprecated("Use of Padded Shape is deprecated")]]
+    static TensorLayout fromPaddedShape(
```

Comment:
Intentional `camelCase` convention to further signal deprecation?

Reply:
Hm, I didn't think about it actually, I was thinking about `fromLegacyPaddedShape` and just made `fromPaddedShape` the same way. But yes, both of them are deprecated, so we can use it as a signal for deprecation.